repoclosure/yum_el6.conf

```diff
@@ -23,6 +23,11 @@ enabled=1
 mirrorlist=http://mirrorlist.centos.org/?release=6&arch=x86_64&repo=updates
 failovermethod=priority
 
+[el6-extras]
+name=extras
+enabled=1
```

Comment:
Please keep this disabled by default, it's a CentOS-only repository and we shouldn't assume it's available usually.